src/plugins/top-bar/components/AboutMenu/AboutMenu.jsx

```diff
@@ -0,0 +1,38 @@
+import PropTypes from 'prop-types';
+
+const AboutMenu = ({ getComponent }) => {
+  const DropdownMenu = getComponent('DropdownMenu');
+  const Link = getComponent('Link');
+
+  return (
+    <>
+      {/* eslint-disable-next-line react/jsx-props-no-spreading */}
+      <DropdownMenu label="About">
+        <li>
```

Comment:
DropdownMenuItem should have the following form:

```jsx
<div>
  <li className="dropdown-item">
    <Link href="https://swagger.io/tools/swagger-editor/" target="_blank">
      About Swagger Editor
     </Link>
  </li>
</div>
```

Can you align the code to reflect this?